frontend/app/components/nav-bar.js

```diff
@@ -0,0 +1,29 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class NavBarComponent extends Component {
+  @tracked
+  isNewAccount = false;
+
+  @tracked
+  isNewFolder = false;
+
+  @tracked
+  isNewTeam = false;
+
+  @action
+  toggleAccountCreating() {
```

Comment:
openAccountCreating ? toggle means open/close ?

Reply:
Which it does, we pass it to the account-form component as the onAbort hook.